packages/create-sitecore-jss/src/common/processes/transform.ts

```diff
@@ -251,12 +252,13 @@ export const transform = async (
   options: TransformOptions = {}
 ) => {
   const { isFileForCopy, isFileForSkip, fileForCopyRegExp = FILE_FOR_COPY_REGEXP } = options;
-  let destination = undefined;
-  // allow proxy app to be installed separately alongside base app
-  if (templatePath.match(/.*node-.+-proxy$/g) && answers.proxyAppDestination) {
-    destination = answers.proxyAppDestination;
-  }
-  const destinationPath = path.resolve(destination || answers.destination);
+  // allow proxy app to be installed and processed separately alongside base app
+  const destination =
```

Comment:
> can we just set the destination in "src/initializers/node-xmcloud-proxy/index.ts"
not really. Templates can be listed in different order during installation. So for example if the command is running like
 > npx create-sitecore-jss templates angular, node-xmcloud-proxy, angular-xmcloud --proxyAppDestination "./not-main-app"
 then the main app destination in args will be overwritten at midway point, this will break the installation.

Reply:
this will be done